### DISCUSSION TITLE
It would be nice to make urls clickable in ray

### DISCUSSION THREAD

**Discussion by vesper8** [spatie/ray#393] [Ideas]
I use iTerm2 for my command prompt and I love the fact that when I press on the CMD key it makes all URLs become clickable. I think this would a nice little addition to ray that I would use often

Maybe you can use this library to identify all urls easily https://github.com/vstelmakh/url-highlight

**Comment by AdrianMrn** [comment]
I like this idea, but I think we'll hold off on it for a bit. We're currently trying to improve the performance of the rendering, so I think adding more steps to the React chain won't help us there.

I'll revisit this one when we've cleared some performance issues off the board :)

**Reply by vesper8** [reply]
Is this board public by any chance? Or is there a public roadmap. I'm really loving Ray and looking forward to improvements

**Reply by AdrianMrn** [reply]
Not right now 😁 but the issue tracker on this repository does partly show what we're working on.
